frontend/src/main/pages/Rider/RiderCreatePage.js

```diff
@@ -0,0 +1,56 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import RiderForm from "main/components/Rider/RiderForm";
+import { Navigate } from 'react-router-dom'
+import { useBackendMutation } from "main/utils/useBackend";
+import { toast } from "react-toastify";
+import { useCurrentUser } from "main/utils/currentUser";
+
+export default function RiderCreatePage() {
```

Comment:
Can we call this a name that matches what the user see and thinks of this as?
```suggestion
export default function RideRequestCreatePage() {
```

I've had a lot of experience with code bases where names in the code are out of alignment with what the  user thinks and sees. Everyone hates working in those code bases.  The best time to fix it is before it goes into the main branch.